src/proposals/20240221/SparkEthereum_20240221.t.sol

```diff
@@ -0,0 +1,75 @@
+// SPDX-License-Identifier: AGPL-3.0
+pragma solidity ^0.8.10;
+
+import '../../SparkTestBase.sol';
+
+interface IIRM {
+    function RATE_SOURCE() external view returns (address);
+}
+
+interface IRateSource {
+    function getAPR() external view returns (int256);
+}
+
+contract SparkEthereum_20240221Test is SparkEthereumTestBase {
+
+    address public constant OLD_DAI_INTEREST_RATE_STRATEGY  = 0x512AFEDCF6696d9707dCFECD4bdc73e9902e3c6A;
+    address public constant NEW_DAI_INTEREST_RATE_STRATEGY  = 0x3C4B090b5b479402e2270C66461D6a62B2054198;
+
+    int256 public constant DAI_IRM_SPREAD =  0.016060808179122167684448000e27;
+
+    uint256 public constant WSTETH_SUPPLY_CAP_OLD = 800_000;
+    uint256 public constant WSTETH_SUPPLY_CAP_NEW = 1_200_000;
+
+    constructor() {
+        id = '20240221';
+    }
+
+    function setUp() public {
+        vm.createSelectFork(getChain('mainnet').rpcUrl, 19233093);  // Feb 15, 2024
+        payload = deployPayload();
+
+        loadPoolContext(poolAddressesProviderRegistry.getAddressesProvidersList()[0]);
+    }
+
+    function testSpellSpecifics() public {
+        ReserveConfig[] memory allConfigsBefore = createConfigurationSnapshot('', pool);
+
+        ReserveConfig memory daiConfigBefore    = _findReserveConfigBySymbol(allConfigsBefore, 'DAI');
+        ReserveConfig memory wstethConfigBefore = _findReserveConfigBySymbol(allConfigsBefore, 'wstETH');
+
+        assertEq(daiConfigBefore.interestRateStrategy, OLD_DAI_INTEREST_RATE_STRATEGY);
+        assertEq(wstethConfigBefore.supplyCap, WSTETH_SUPPLY_CAP_OLD);
+
+        GovHelpers.executePayload(vm, payload, executor);
+
+        ReserveConfig[] memory allConfigsAfter = createConfigurationSnapshot('', pool);
+
+        ReserveConfig memory daiConfigAfter     = _findReserveConfigBySymbol(allConfigsAfter, 'DAI');
+
+        int256 potDsrApr = IRateSource(IIRM(daiConfigAfter.interestRateStrategy).RATE_SOURCE()).getAPR();
```

Comment:
Before that, I'd also check that `IIRM(daiConfigAfter.interestRateStrategy).RATE_SOURCE()` is equal to a proper hardcoded address

Reply:
Checked on etherscan, it's ok, but I'd still test it just for clarity

Reply:
I check that it's the same as before. That seems sufficient to me, but lmk if you want hardcoded address.

Reply:
Ah yes, I missed it. Yeah, I think it's sufficient.